app/utils/jspsych/functions.js

```diff
@@ -12,6 +13,22 @@ import {
   ExperimentParameters
 } from '../../constants/interfaces';
 
+// create and open an outlet to stream data
+const info = lsl.create_streaminfo(
+  'Dummy',
+  'Marker', // type
+  1, // number of channels
+  10, // sample rate in 1 sec.
+  lsl.channel_format_t.cft_float32,
+  'Dummy Marker Device'
+);
+const desc = lsl.get_desc(info);
+lsl.append_child_value(desc, 'manufacturer', 'Random Inc.');
+const channels = lsl.append_child(desc, 'channels');
+const channel = lsl.append_child(channels, 'channel');
+lsl.append_child_value(channel, 'type', 'Marker');
+const outlet = lsl.create_outlet(info, 0, 360);
```

Comment:
just curious, what does the `360` refer to?

Reply:
In node-lsl documentation, it's "max_buffered"
https://github.com/urish/node-lsl/blob/master/index.d.ts#L50

Reply:
got it, thanks